src/queue/Playlist.hxx

```diff
@@ -114,7 +114,7 @@ struct playlist {
 	int GetCurrentPosition() const noexcept;
 
 	gcc_pure
-	int GetNextPosition() const noexcept;
+	int GetNextPosition(unsigned index) const noexcept;
```

Comment:
Please document this - the method is currently completely undocumented, that's bad already (sorry...)...